http/codegen/openapi/openapi_v2_builder.go

```diff
@@ -562,6 +562,15 @@ func buildPathFromExpr(s *V2, root *httpdesign.RootExpr, route *httpdesign.Route
 
 		responses := make(map[string]*Response, len(endpoint.Responses))
 		for _, r := range endpoint.Responses {
+			if endpoint.MethodExpr.IsStreaming() {
+				// A streaming endpoint allows at most one successful response
+				// definition. So it is okay to change the first successful
+				// response to a HTTP 101 response for openapi docs.
```

Comment:
Can a 101 response contain a body? if not seems we should also delete the body if present?

Reply:
Doesn't really say anywhere that 101 cannot have a body.